packages/ucp/.gitignore

```diff
@@ -0,0 +1 @@
+debug
```

Comment:
What produces the `debug` file/folder?

Reply:
https://github.com/SolidLabResearch/user-managed-access/blob/feat/ucp-migration/packages/ucp/test/util/Validation.ts#L36

This function is currently called in the testing environment.

In other words, if changes are made to the code and the current tests (due to the methods `validate` and `validateAndExplain`) fail, then files are added to a debug folder. And you don't want those files on github.

This can be removed if we have proper unit and integration tests